### ISSUE TITLE
Job are submitted but not started.

### ISSUE THREAD

**Issue by tmoulin** [ow2-proactive/scheduling-portal#310]
In the Scheduler, jobs are submitted (via "Submit job") but are not executed and do not appear.
Observed when submitting variables-less workflows imported from catalog.
Works fine when importing from file, even if said file is from the catalog.
Works fine when using a workflow that was added to the catalog via the studio even if variables-less.

**Comment by tobwiens** [comment]
Hello,

That's sounds weird. There might be a configuration problem. 
Can you provide the logs? You find them here: logs/Scheduler.log

**Comment by tmoulin** [comment]
Thanks.

Here are the logs of a few attempts.

[Log.txt](https://github.com/ow2-proactive/scheduling-portal/files/424803/Log.txt)

**Comment by marcocast** [comment]
@tobwiens No particular information or errors in the logs.

**Comment by marcocast** [comment]
I suspect it s the template that is somehow wrong and when it s imported in the studio, the studio fix its structure automagically..

**Comment by tobwiens** [comment]
The Log.txt looks cut. Are you sure that it is the whole file? I don't see any log entries after the workflow xml submission. That looks unusual. 

Can you re-download Workflows & Scheduling and try again. Then send the logs again. 
Thank you.

**Comment by tmoulin** [comment]
Sorry, didn't check.
This should contain the information you need. 
[Log.txt](https://github.com/ow2-proactive/scheduling-portal/files/426970/Log.txt)

**Comment by tobwiens** [comment]
I can see in the logs, that you submitted several javascript hello world workflows. Are those workflows executed? 

I didn't see any error message indicating that a wrong Worklfow submitted and not executed. 

Can you send me a screenshot of the Scheduler interface after you submitted those jobs? It might be that a filter is set so that you don't see those jobs. 

Thank you,
Tobias

**Comment by tmoulin** [comment]
Hello.

As far as I can tell, the workflows were not executed.

Here are screenshots on try (7.15.0) and local (7.15.0-20160818-0411-SNAPSHOT).
![try](https://cloud.githubusercontent.com/assets/20511930/17847018/049d7240-684c-11e6-9c79-eb1149245703.jpg)
![local](https://cloud.githubusercontent.com/assets/20511930/17847020/07b38fe6-684c-11e6-943e-84533445efad.png)

Thank you.
Thomas.

**Comment by tobwiens** [comment]
The screenshots are from try.activeeon.com. Are you working on try.activeeon.com or are you working with a Scheduler deployed by yourself? 

I checked try.activeeon.com and the Workflow execution with the JavaScript "Hello World" works. 

If you use your own deployment, then you need to go to your local web portal localhost:8080/scheduler or the correct IP or Hostname instead of localhost.

**Comment by tmoulin** [comment]
The first screenshot is from try.activeeon.com but the second one is from a scheduler I deployed.

The problem does not occur when submitting the workflow from the studio, only when importing one of the workflows in the catalog and submitting it, all from the scheduler portal.

**Comment by tobwiens** [comment]
Okay thank you very much for explaining it precisely. I see exactly what you mean. 

**Problem description**
A workflow is not executed/submitted/does not appear in the Scheduler interface when 

- Submitted over the Scheduler portal
- From the catalog
- Has no variables 

**Reproducing the problem:**
1. Importing a workflow from the catalog (which has no variables)
![image](https://cloud.githubusercontent.com/assets/6491363/17883544/79abde60-6913-11e6-8a02-488be5da19c1.png)

2. Submitting the workflow
![image](https://cloud.githubusercontent.com/assets/6491363/17883528/5d743152-6913-11e6-9bb1-53ace1348b9c.png)

3. Workflow is not executed
![image](https://cloud.githubusercontent.com/assets/6491363/17883559/8c5c47b6-6913-11e6-9dda-2a30f85bfc38.png)



**It is working when the workflow has variables**
1. Importing a workflow from the catalog
![image](https://cloud.githubusercontent.com/assets/6491363/17883581/a6659c5c-6913-11e6-8e91-04fefd989203.png)

2. Submitting the workflow
![image](https://cloud.githubusercontent.com/assets/6491363/17883592/b64a5af4-6913-11e6-98f8-b6b323cd2a20.png)

3. Workflow is executed
![image](https://cloud.githubusercontent.com/assets/6491363/17883619/e8babe66-6913-11e6-8e9f-7582107ad079.png)


@tmoulin Thank you very much! Do you have something to add to the problem description?

**Comment by tmoulin** [comment]
Yes, when, in the Studio, a workflow is added to the catalog (after being created or imported from the catalog), the workflow can be executed without problems, as if it had variables. (i.e: only the default workflows are affected => schema problem?)

**Comment by tobwiens** [comment]
@tmoulin Thank you, for adding that. I also added a point in the above description to make it clear for us. 
Maybe it could be a schema, that's a good idea. 

Maybe the Schema is not updated to the current schema 3.5 and that might cause some issues.

**Comment by tmoulin** [comment]
I just checked and indeed the Schema is 3.3.

**Comment by tobwiens** [comment]
@tmoulin If you like you can submit a patch/ or test it yourself. 
Do you feel like doing it? What is your experience level?

**Comment by tmoulin** [comment]
@tobwiens  Sorry, I do not think I have the level to do it.
